OpenSim/Moco/MocoFrameOrientationConstraint.h

```diff
@@ -0,0 +1,80 @@
+#ifndef OPENSIM_MOCOFRAMEORIENTATIONCONSTRAINT_H 
+#define OPENSIM_MOCOFRAMEORIENTATIONCONSTRAINT_H 
+/* -------------------------------------------------------------------------- *
+ * OpenSim Moco: MocoFrameOrientationConstraint.h                             *
+ * -------------------------------------------------------------------------- *
+ *                                                                            *
+ * Author(s): Size Zheng                                              		  *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0          *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include "MocoConstraint.h"
+#include <OpenSim/Moco/osimMoco.h>

```

Comment:
I don't think this include is necessary since you already have `osimMocoDLL.h` included below.